### PR TITLE
Get status messages from exit file

### PR DESCRIPTION
**Pull request by eivindjahren**
## Pre review checklist

- [x] Added appropriate release note label
- [x] PR title captures the intent of the changes, and is fitting for release notes.
- [x] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).
- [ ] Updated documentation
- [ ] Ensured new behaviour is tested

Adding labels helps the maintainers when writing release notes. This is the [list of release note labels](https://github.com/equinor/ert/labels?q=release-notes).

### CODE REVIEW

File: src/clib/lib/job_queue/job_node.cpp [hunk 1]
```diff
@@ -137,21 +130,26 @@ static char *__alloc_tag_content(const char *xml_buffer, const char *tag) {
...
+                fmt::format("max_confirm_wait ({}) has passed since sim_start"
+                            "without success; {} is assumed dead (attempt {})");
 
             free(xml_buffer);
-        } else
-            node->failed_job = util_alloc_sprintf(
-                "EXIT file:%s not found - load failure?", node->exit_file);
+        }
+        node->fail_message = fmt::format(
+            "EXIT file:{} not found - load failure?", node->exit_file);
```
[the remark this answers is not in the document]
Comment (eivindjahren) [reply src/clib/lib/job_queue/job_node.cpp]: Both job_queue_node.py and queue_node.cpp call this the EXIT file. perhaps we can have a separate PR with this rename?

File: src/ert/gui/ertwidgets/message_box.py [hunk 2]
```diff
@@ -7,6 +7,11 @@
     QTextEdit,
 )
 
+from ..tools.file.file_dialog import (
```
Comment (eivindjahren) [inline src/ert/gui/ertwidgets/message_box.py:10]: Should probably absolute import instead of `..`

**Comment by valentin-krasontovitsch** [comment]
close in favor of #5841 which contains tests?

**Comment by eivindjahren** [comment]
Superseded by #5841

**Pull request closed without merging**